packages/packages/react-native-sdk/metriport-react-native.podspec

```diff
@@ -0,0 +1,37 @@
+require "json"
+
+package = JSON.parse(File.read(File.join(__dir__, "package.json")))
+folly_compiler_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32'
+
+Pod::Spec.new do |s|
+  s.name         = "metriport-react-native"
+  s.version      = package["version"]
+  s.summary      = package["description"]
+  s.homepage     = package["homepage"]
+  s.license      = package["license"]
+  s.authors      = package["author"]
+
+  s.platforms    = { :ios => "13.0" }
+  s.source       = { :git => "https://github.com/metriport.git", :tag => "#{s.version}" }
```

Comment:
`https://github.com/metriport/metriport.git`  ?

Reply:
I forgot an extra metriport but this is the url https://github.com/metriport/metriport.git it just reroutes to the repo but that seems to be the standard

https://guides.cocoapods.org/syntax/podspec.html#source